### ISSUE TITLE
I'm getting a bug where recoverPoints resets to 5000 everytime I die.

### ISSUE THREAD

**Issue by geralddzx** [games50/breakout#1]
https://github.com/games50/breakout/blob/d9b65b96eafb90067f8cd4b47e3f6799bee88493/breakout13/src/states/PlayState.lua#L32

**Comment by geralddzx** [comment]
I think this should be.  `self.recoverPoints = params.recoverPoints`

**Comment by fontinixxl** [comment]
I agree with you @geralddzx I figured out the same and that is the way I fixed as well.

**Comment by lstanard** [comment]
You need to have an initial value of some sort otherwise you can never recover health as params.recoverPoints is initially nil

`self.recoverPoints = params.recoverPoints and params.recoverPoints or 5000`

**Comment by edischin** [comment]
Initial value can be set in PaddleSelectState when it changes to ServeState, beginning a new game. PlayState would then receive recoverPoints in params, as mentioned by @geralddzx and @fontinixxl .
